pages/windows/pushd.md

```diff
@@ -0,0 +1,7 @@
+# pushd
+
+> Place a directory on a stack so it can be accessed later.
```

Comment:
Add another line as subheading like - 
"Also look at `popd` to switch back to original directory"

Reply:
Or even `See also` instead of `Also look`